audit/org_kubernetes.io/roles/audit.viewer.json

```diff
@@ -145,6 +145,100 @@
     "binaryauthorization.policy.getIamPolicy",
     "clientauthconfig.brands.list",
     "clientauthconfig.clients.list",
+    "cloudasset.assets.analyzeIamPolicy",
```

Comment:
All permission changes here expected, part of https://github.com/kubernetes/k8s.io/pull/2010 being deployed (https://github.com/kubernetes/k8s.io/pull/2010/commits/6d93372fab03a8e71db30d0d684e932deaad733e)